pkg/profiler/profiler.go

```diff
@@ -303,6 +304,34 @@ func (p *CgroupProfiler) Run(ctx context.Context) error {
 	ctx, p.cancel = context.WithCancel(ctx)
 	p.mtx.Unlock()
 
+	isContainer, err := kconfig.IsInContainer()
+	if err != nil {
+		level.Warn(p.logger).Log("msg", "Check container error:", err)
+	}
+
+	if isContainer {
+		level.Info(p.logger).Log("msg", "Parca agent is running in a container. It'll need to access the host kernel config")
+	}
+
+	bpfEnabled, err := kconfig.IsBPFEnabled()
+	if err != nil {
+		level.Warn(p.logger).Log("msg", "failed to determine if eBPF is supported", "err", err)
```

Comment:
Let's start with warnings as I'm sure this doesn't cover all the possible cases where kconfig file would be found. I'll send a follow-up PR for it in the upcoming weeks. We can error out after that.